new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -157,9 +160,20 @@
    * @return true if the pod is terminating otherwise false
    */
   public static Callable<Boolean> podTerminating(String podName, String domainUid, String namespace) {
-    return () -> Kubernetes.isPodTerminating(namespace, domainUid, podName);
+    return Pod.podTerminating(namespace, domainUid, podName);
   }
 
+  /**
+   * Check pods are restarted in the same order as in the pods list.
+   * @param pods names of the pods in a list
+   * @param namespace name of the namespace in which to check for pods rolling restart
+   * @return true if pods are restarted in rolling fashion
+   */
+  public static boolean verifyRollingRestartOccurred(ArrayList<String> pods, String namespace) {
```

Comment:
Type of argument can just be List<String>

Reply:
Now the argument is a Map with podnames and its creation time stamps.
The test will create a LinkedHashMap and add all the pods with its creation time stamps.